rpcs3/util/fixed_typemap.hpp

```diff
@@ -241,6 +261,38 @@ namespace stx
 			}
 		}
 
+		void stop_all()
```

Comment:
Instead need an iterator over existing objects exporting typeinfo data, or it'll become quite a mess.

Reply:
Fixed.

Reply:
Not fixed, it's not what I meant. It should export typeinfo so stop_all is not written inside here.